index.js

```diff
@@ -53,34 +54,31 @@ function Github (opts) {
   }
 
   function _create (key, opts, cb) {
-    var code = opts.code
-    _oauth(code, function (user) {
-      var res = {
-        username: user.login
-      }
+    assert.equal(typeof opts, 'object', 'township-github._create: opts should be type Object')
 
-      cb(null, res)
-    })
+    var res = {
+      username: opts.username
+    }
+    cb(null, res)
   }
 
   function _verify (opts, cb) {
```

Comment:
@yoshuawuyts there is somethig fishy going on in `_verify` --  it sometimes comes back with 500 when it shouldn't. so like mackerel kind of fish, not even salmon. holla if you have ideas o/